ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -55,6 +55,36 @@ code change.
      - Bare-metal
      - \-
 
+Quality managed targets
+-----------------------
+
+Quality managed targets are targets which are suitable for use in production 
+outside of safety critical contexts. Support is available. The Ferrocene test
+suite successfully executed on the target. Known problems are tracked and
+available.
+
+For any of the following reasons, the target is not qualified:
+
+* The target is deemed unlikely to be part of a safety critical context.
```

Comment:
would it be more clear (and retain same meaning) if we said *to be used in a safety-critical context*